main.qml

```diff
@@ -52,68 +151,146 @@ ApplicationWindow {
 				onClicked: vision.calibrationRunning = true;
 			}
 		}
-	}
+    }
 
-	Camera {
+    Camera {
 		id: camera
 
-		focus {
-			focusMode: Camera.FocusAuto
-		}
+        // For the logitec c920 webcam the following viewfinder resolutions work:
+        //  "640x480" / "1280x720" / "1600X986" / "1920x1080" */
+        viewfinder.resolution: "1280x720"
 
-		captureMode: Camera.CaptureViewfinder
+        captureMode: Camera.CaptureViewfinder
 		cameraState: Camera.LoadedState
-		//deviceId: QtMultimedia.availableCameras[1].deviceId // hack to use second camera on laptop
+
+        //deviceId: QtMultimedia.availableCameras[1].deviceId // hack to use second camera on laptop
 	}
 
+    // HACK
```

Comment:
Why? If it is a hack, why is it still here? If it works around a bug in Qt, then please explain in detail why this is here.